packages/fast-sketch-library/readme.md

```diff
@@ -0,0 +1,40 @@
+# FAST-DNA sketch library
+`@microsoft/fast-sketch-library` is a tool for quickly extracting sketch libraries from a website. It uses [https://github.com/brainly/html-sketchapp/](https://github.com/brainly/html-sketchapp/) to create an "almost sketch" JSON file which can be loaded into sketch using the plugin provided by `html-sketchapp`.
+
+## Usage
+Install the package
+`npm i -D @microsoft/fast-sketch-library`
+
+Set up a node script to extract symbols from a library
+
+```js
+const extractSymbolLibrary = require("@microsoft/fast-sketch-library").extractSymbolLibrary;
+const fs = require("fs");
+
+const config = {
+    name: "Fast-dna MSFT design kit",
```

Comment:
Capitalize FAST-DNA

Reply:
actually changed this because I think it is better if it generic